SpinnerDatePickerLib/src/main/java/com/tsongkha/spinnerdatepicker/DatePickerDialog.java

```diff
@@ -26,6 +26,9 @@
     private final OnDateSetListener mCallBack;
     private final DateFormat mTitleDateFormat;
 
+    private boolean mIsDayEnabled = true;
+    private boolean mUpdateTitleEnabled = true;
```

Comment:
Can we please call this `mShowTitle` since setting it to false completely hides the title?